CMakeLists.txt

```diff
@@ -18,17 +18,11 @@ if(NOT avk_LibraryType IN_LIST avk_AllowedLibraryTypes)
     message(FATAL_ERROR "avk_LibraryType must be one of ${avk_AllowedLibraryTypes}")
 endif()
 
-option(avk_UseVMA "Use Vulkan Memory Allocator (VMA) for custom memory allocation." OFF)
```

Comment:
It is possible to build Auto-Vk as a static or shared library. In such cases, since `AVK_USE_VMA` needs to be defined when compiling the library, it might be a good idea to keep the CMake option and add define `AVK_USE_VMA` if `vk_UseVMA` is turned on (I think via `CMAKE_CXX_FLAGS`?).
Maybe not, I don't have a strong opinion on this.
In any case, it would be good to test for building with and without VMA in the GH actions.

Reply:
I agree.. good idea. Do you know from the top of your head how to turn that into CMake?

Reply:
I just googled it. I think it would be
```
if(avk_UseVMA)
    add_compile_definitions(AVK_USE_VMA)
endif()
```

Reply:
And in the GH action it would be something like
```
cmake -Davk_UseVMA=ON ...
```
Testing for both, ON and OFF, can be configured via the job's strategy (in the `matrix`)

Reply:
I think, I managed to configure it, but we have the following problem: 
``` 
static_assert(false, "Unable to find include <vma/vk_mem_alloc.h> or <vk_mem_alloc.h>, but AVK_USE_VMA is defined.");
``` 

And that's because the SDK is used without having "installed" VMA. Any way to request a Vulkan SDK which comes _with_ VMA?

Reply:
Ah, on windows right? I found this action we could use to install the sdk instead of doing it ourselves (we currently just download it to a random location on windows, right?): https://github.com/marketplace/actions/install-vulkan-sdk

That won't help for version 1.2.xxx though, because it doesn't include vma.